lineage/import_lineage.py

```diff
@@ -185,7 +172,10 @@ def assign_observs_MCF10A(cell, lineage: list, uniq_id: int):
 
     # cell's lifetime
     cell.obs[1] = (np.max(lineage.loc[lineage['TID'] == uniq_id]['tmin']) - np.min(lineage.loc[lineage['TID'] == uniq_id]['tmin'])) / 60
-
+    cell.obs[3] = np.mean(lineage.loc[lineage['TID'] == uniq_id]['average_velocity'])
+    cell.obs[4] = np.mean(lineage.loc[lineage['TID'] == uniq_id]['distance_mean'])
+    if cell.obs[1] <= 0.1:
+        cell.obs[1] = 0.5
```

Comment:
Is this necessary? Why are there cells with lifetimes less than 0.1?

Reply:
There were the cells with 0 lifetime, I fixed it now.